### PR TITLE
feat: Add support from hidden suppressions migration command

### PR DESCRIPTION
**Pull request by rmoles**
Signed-off-by: Ross <ross.moles@lacework.net>

## Summary

As part of the legacy compliance deprecation effort, the legacy cfg-analyzers will be decommissioned some time in January 2023. As a result customers will be moved across to the new LPP policies and CIS benchmarks. This is likely to cause a lot of noise for customers as they will have suppressions in place for resources which they expect to be in a non compliant state.
At present there is no easy path for customers to migrate their old suppressions to the new exceptions policies.

We do have a scripted approach created by Marc Garcia: https://github.com/lacework-dev/scripts/tree/main/lw-suppressions-migration

However it was decided that we should add this to the CLI as a more "official" approach.

This PR adds the following hidden commands:

```
lacework suppressions list aws
lacework suppressions migrate aws
```
suppressions has the following alias' `suppression, sup, sups`
list has the following alias `ls`
migrate has the following alias `mig`

`lacework suppressions list`
Will allow the users to get their legacy suppressions. Currently they aren't able to view these anywhere else...

`lacework suppressions migrate aws`
Will enable users to migrate their legacy suppressions to new LPP policy Exceptions.
This supports the migration in 2 modes:
 - Manual (recommended): This outputs the converted suppressions as commands that the user can review and run as they wish
 - Automatic: This gives the user the option to allow Lacework to handle the suppressions migration. It will output the converted legacy suppressions in the Exception format and ask the user to confirm they are happy with the Exceptions and accepting liability for any Compliance violations that may be missed as a result of this.

#1066 & #1068 should be merged before this

## How did you test this change?
Tested locally ✅ 

## Issue
https://lacework.atlassian.net/browse/45215

### CODE REVIEW

File: cli/cmd/suppressions.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+//
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	// top-level cloud-account command
```
Comment (afiune) [inline cli/cmd/suppressions.go:26]: this needs an update

File: cli/cmd/suppressions.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	suppressionsCommand = &cobra.Command{
+		Use:     "suppressions",
+		Hidden:  true,
+		Aliases: []string{"suppression", "sup", "sups"},
```
Comment (afiune) [inline cli/cmd/suppressions.go:30]: do we really need aliases like `sup` and `sups`? if this is a hidden command I don't think we do need so many aliases.

Comment (afiune) [reply cli/cmd/suppressions.go:30]: But I am ok leaving them 😂 this is a nit

**Review by afiune — CHANGES_REQUESTED** [review]
If the Lacework account does not have any AWS account configured, the list command shows the following error message:
```
ERROR unable to get legacy aws suppressions:
  [GET] https://customerdemo.lacework.net/api/v2/suppressions/aws/allExceptions
  [400] Cannot perform operation. No active AWS accounts
```

Can we improve this and make it just a message without error saying that there are no AWS accounts configured.

**Comment by afiune** [comment]
Also, I feel that the commands should be flipped around, what do you think @rmoles ??
```
$ lacework suppressions aws list
$ lacework suppressions aws migrate

// in the future
$ lacework suppressions gcp list
$ lacework suppressions gcp ....
```

File: cli/cmd/suppressions.go (hunk 2 above)
Comment (rmoles) [reply cli/cmd/suppressions.go:30]: probably not, but a the same time, why not? 🤷

**Review by afiune — APPROVED** [review]
This is way too much code for just one single test, I would highly recommend writing more tests, both unit tests (for functional testing) and integration tests (for end-to-end tests).

Code looks Ok.

**Pull request merged**